src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileBlock.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (nos-devel@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.util.partyline.impl.infinispan.model;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.nio.ByteBuffer;
+import java.util.Date;
+import java.util.UUID;
+
+public class FileBlock
+        implements Externalizable
+{
+    private String fileID;
+
+    private UUID blockID;
```

Comment:
Change these to String representations of the UUIDs, since UUID.fromString(..) doesn't reproduce something that works with Object.equals()